core/camel-core-reifier/src/main/java/org/apache/camel/reifier/ResumableReifier.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.camel.reifier;
+
+import org.apache.camel.Processor;
+import org.apache.camel.ResumeStrategy;
+import org.apache.camel.Route;
+import org.apache.camel.model.ProcessorDefinition;
+import org.apache.camel.model.ResumableDefinition;
+import org.apache.camel.processor.resume.ResumableProcessor;
+import org.apache.camel.util.ObjectHelper;
+
+public class ResumableReifier extends ExpressionReifier<ResumableDefinition> {
+    protected ResumableReifier(Route route, ProcessorDefinition<?> definition) {
+        super(route, ResumableDefinition.class.cast(definition));
+
+    }
+
+    @Override
+    public Processor createProcessor() throws Exception {
+        Processor childProcessor = createChildProcessor(false);
+
+        ResumeStrategy resumeStrategy = resolveResumeStrategy();
+        ObjectHelper.notNull(resumeStrategy, "resumeStrategy", definition);
+        resumeStrategy.start();
```

Comment:
Do not start in create, as reifier should ideally only create, and start is done later when Camel starts the routes

Reply:
And let ResumableProcessor handle the lifecycle where it can start the strategy from its doInit / doStart methods